app/views/collections/_document.html.erb

```diff
@@ -11,27 +20,50 @@
           <div class="pb-4 mr-2">
             <%= link_to title_for(document), collection_document_path(document.collection, document), format: :turbo_stream %>
           </div>
-          <%= render "shared/document_state_badges", document: %>
+          <% if document.no_children? %>
+            <%= render "shared/document_state_badges", document: %>
+          <% end %>
         </div>
       </div>
-      <div class="flex">
-        <%= link_to "View", collection_document_path(document.collection, document), class: "rounded-lg bg-secondary-700 dark:bg-secondary-400 text-secondary-100 dark:text-secondary-900 py-1 px-4 disabled:bg-opacity-40 mr-2" %>
-        <%= button_to "Reprocess", collection_document_vectorize_path(document.collection, document), class: "rounded-lg px-4 py-1 text-center text-white align-baseline dark:text-secondary-950 block font-medium bg-secondary-700 dark:bg-secondary-400 mr-2" %>
-        <% if document.stop_jobs %>
-          <%= button_to "Enable jobs", collection_document_start_path(document.collection, document), class: "rounded-lg px-4 py-1 text-center text-white align-baseline dark:text-secondary-950 block font-medium bg-secondary-700 dark:bg-secondary-400" %>
-        <% else %>
-          <%= button_to "Disable jobs", collection_document_stop_path(document.collection, document), class: "rounded-lg px-4 py-1 text-center text-white align-baseline dark:text-secondary-950 block font-medium bg-secondary-700 dark:bg-secondary-400" %>
+
+      <!-- Show the document-specific actions -->
+      <% if document.no_children? %>
+        <div class="flex">
+          <%= link_to "View", collection_document_path(document.collection, document), class: "rounded-lg bg-secondary-700 dark:bg-secondary-400 text-secondary-100 dark:text-secondary-900 py-1 px-4 disabled:bg-opacity-40 mr-2" %>
+          <% if document.stop_jobs %>
+            <%= button_to "Enable jobs", collection_document_start_path(document.collection, document), class: "rounded-lg px-4 py-1 text-center text-white align-baseline dark:text-secondary-950 block font-medium bg-secondary-700 dark:bg-secondary-400" %>
+          <% else %>
+            <%= button_to "Disable jobs", collection_document_stop_path(document.collection, document), class: "rounded-lg px-4 py-1 text-center text-white align-baseline dark:text-secondary-950 block font-medium bg-secondary-700 dark:bg-secondary-400" %>
+          <% end %>
+        </div>
+      <% end %>
+
+      <% unless document.no_children? %>
```

Comment:
Not a big deal, but feels like a double-negative.

I try to name methods on objects in a positive way, like `has_children?` to avoid this kind of thing. You could even go with `childless?`.

Reply:
I've replaced that one with `if document.children.any?` to avoid the double negative.